playwright/playwright.config.ts

```diff
@@ -26,10 +26,9 @@ reporter.push(
  */
 const config: PlaywrightTestConfig = {
     testDir: pathFromRoot('src'),
-    testMatch: '*/__tests__/*.visual.test.tsx',
+    testMatch: '*/__snapshots__/*.visual.test.tsx',
```

Comment:
This should remain `__tests__`, my comment was for `snapshotPathTemplate` line only